src/lib/components/menu/compositions/SelectMenu.svelte

```diff
@@ -0,0 +1,72 @@
+<!-- @component
+A simple menu of actions. When one is clicked, the callback fires,
+and the menu closes.
+
+This component is a fully-opinionated way of using selects.
+A slot is provided to change the text within the button.
+
+-->
+<script lang="ts">
+  import { createEventDispatcher, setContext } from "svelte";
+
+  import { SelectButton, WithSelectMenu } from "../";
+
+  export let options;
+  export let selections = [];
+
+  export let tailwindClasses = undefined;
+  export let activeTailwindClasses = undefined;
```

Comment:
Does this not support multi select as of now? I remember seeing an old dev route with multi select as well.